homeassistant/components/light/lutron.py

```diff
@@ -0,0 +1,94 @@
+"""Support for Lutron lights."""
+import logging
+
+from homeassistant.components.light import ATTR_BRIGHTNESS, Light
+from homeassistant.components.lutron import (
+    LutronDevice, LUTRON_DEVICES, LUTRON_GROUPS, LUTRON_CONTROLLER)
+from homeassistant.util.async import (
+    run_coroutine_threadsafe)
+
+DEPENDENCIES = ['lutron']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+# pylint: disable=unused-argument
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup Lutron lights."""
+    area_devs = {}
+    devs = []
+    for (area_name, device) in hass.data[LUTRON_DEVICES]['light']:
+        dev = LutronLight(hass, area_name, device,
+                          hass.data[LUTRON_CONTROLLER])
+        area_devs.setdefault(area_name, []).append(dev)
+        devs.append(dev)
+    add_devices(devs)
+
+    for area in area_devs:
+        if area not in hass.data[LUTRON_GROUPS]:
+            continue
+        grp = hass.data[LUTRON_GROUPS][area]
+        ids = list(grp.tracking) + [dev.entity_id for dev in area_devs[area]]
+        run_coroutine_threadsafe(
```

Comment:
fixed up calling async properly, I think? Sorry, not very familiar with all this python asyncio stuff.